app/javascript/packs/application.js

```diff
@@ -3,6 +3,9 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
+//= require jquery3
```

Comment:
is this similar to `require("jquery")` ? could you please confirm if your code is working?